tests/unit_tests/data_model/test_model_data_writer.py

```diff
@@ -12,6 +12,79 @@
 logger.setLevel(logging.DEBUG)
 
 
+def test_write(tmp_test_directory):
+    # both none (no exception expected)
+    w_1 = writer.ModelDataWriter()
+    w_1.write(metadata=None, product_data=None)
```

Comment:
According to the function `write` this is supposed to have no effect, right? But it seems to me that in this test we are not checking/asserting anything. maybe we should check that no file was written?

Reply:
But this is obvious and not needed to be tested.

What important is in this case (both none) that no Exception is raised.